src/abhishekSJun20/CheckpalindromeString.java

```diff
@@ -0,0 +1,22 @@
+package abhishekSJun20;
+
+public class CheckpalindromeString {
+	// method to check String is palindrome or not
+	void palindromeStringCheck(String input) {
+		String rev = "";
+		for (int index = input.length() - 1; index >= 0; index--) {
+			rev = rev + input.charAt(index);
+		}
+		if (input.equalsIgnoreCase(rev))
```

Comment:
You should always use equals() method for palindrome. Because Radar is not Palindrome string.